hw/usb/tinyusb/da146xx/include/tusb_hw.h

```diff
@@ -22,9 +22,28 @@
 
 #define CFG_TUSB_MCU OPT_MCU_DA1469X
 
+#include <os/mynewt.h>
 #include <syscfg/syscfg.h>
 #include <stdbool.h>
 
+#if MYNEWT_VAL(USB_MONITOR)
+/**
+ * Set event queue for USB monitoring
+ */
+void tinyusb_usb_monitor_evq_set(struct os_eventq *evq);
+
+/**
+ *
+ * Register callback for USB state monitor changes
+ */
+void tinyusb_usb_monitor_register_cb(void (*cb)(bool connected));
+#endif
+
+/**
+ * Check if USB is connected (based on keep-alive and VBUS if used)
+ */
+bool tinyusb_is_connected(void);
```

Comment:
does this need to be public (and why in da146xx header?) ?  it seems to be used only internally by da1469x.c